### PR TITLE
Add Void Linux installation instructions.

### PR DESCRIPTION
**Pull request by ke7ofi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -175,4 +176,4 @@ The current avaliable styles are:
   - The digaphs defined in Unicode is now single-width.
   - Reorganized subfamilies: Now the default Iosevka does not contain full-width glyphs any more.
 
-![Family Matrix](https://raw.githubusercontent.com/be5invis/Iosevka/master/images/matrix.png)
```
Comment (ke7ofi) [inline README.md]: not sure what happened here

**Comment by be5invis** [comment]
I cannot merge your PR directly. I will include it on my own.

**Comment by ke7ofi** [comment]
Do you know what the problem was?

**Comment by be5invis** [comment]
@ke7ofi maybe just an accident.

**Comment by ke7ofi** [comment]
Neat, thanks.

**Pull request merged**